fuzzing/fuzzer.go

```diff
@@ -405,7 +406,22 @@ func chainSetupFromCompilations(fuzzer *Fuzzer, testChain *chain.TestChain) erro
 
 				// Ensure our transaction succeeded
 				if block.MessageResults[0].Receipt.Status != types.ReceiptStatusSuccessful {
-					return fmt.Errorf("contract deployment tx returned a failed status: %v", block.MessageResults[0].ExecutionResult.Err)
+					// Create a call sequence element to represent the failed contract deployment tx
+					cse := calls.NewCallSequenceElement(nil, msg, 0, 0)
```

Comment:
I may have abused the call sequence here as it seems mostly used for "test" transactions and not the target deployment, but this was straightforward to get the execution traces. I think it is fine to leave target contract as `nil` as it makes sense for a deployment, but would appreciate review